packages/react-liveness/src/components/FaceLivenessDetector/shared/LivenessIconWithPopover.tsx

```diff
@@ -31,32 +32,44 @@ export const LivenessIconWithPopover: React.FC<LivenessIconWithPopoverProps> =
         }
       }
       document.addEventListener('mousedown', handleClickOutside);
+
       return () => {
         document.removeEventListener('mousedown', handleClickOutside);
       };
     }, [wrapperRef, shouldShowPopover]);
 
     return (
-      <Flex
-        className={LivenessClassNames.Popover}
-        onClick={() => setShouldShowPopover(!shouldShowPopover)}
-        ref={wrapperRef}
-        testId="popover-icon"
-      >
-        <AlertIcon ariaHidden variation="info" />
-        {shouldShowPopover && (
-          <>
-            <Flex className={LivenessClassNames.PopoverAnchor} />
-            <Flex className={LivenessClassNames.PopoverAnchorSecondary} />
-            <Flex
-              className={LivenessClassNames.PopoverContainer}
-              left={isMobileScreen ? -190 : -108}
-              data-testid="popover-text"
-            >
-              {children}
-            </Flex>
-          </>
-        )}
+      <Flex ref={wrapperRef}>
+        <Button
+          aria-controls="popover-text"
```

Comment:
This needs to reference an `id` that's on the same element as `role="dialog"` We're basically saying "this button controls the showing and hiding of this element"